Complete overhaul of frontend/cleanup

LudvigHz opened the pull request:
This PR bumps a lot of stuff, removes even more stuff and fixes some stuff :shipit: 

- Remove a bunch of deps, and clean up. The same dependencies were installed with cdns and npm :upside_down_face:.
- Bump AdminLTE to v3 for a more modern look.
- Remove webpack, bacause it's useless. The only thing it did was copy some files from node_modules during build. So it was replaced with a simple script.
- Some minor tweaks to the interface.
- General cleanup of the codebase


Resolves #109 (somewhat)

_Also:_
```
yarn audit v1.22.4
0 vulnerabilities found - Packages audited: 233
Done in 0.79s.
```
:fire: :fire:

odinuge commented on package.json:
```diff
@@ -22,29 +20,22 @@
+    "bootstrap-table": "^1.16.0",
+    "chosen-js": "^1.8.7",
     "eonasdan-bootstrap-datetimepicker": "^4.17.47",
     "file-loader": "^3.0.1",
```
:bomb: :radioactive:

LudvigHz replied:
Hmm, I seem to have missed some unused stuff here :open_mouth:

odinuge commented on README.md:
```diff
@@ -86,6 +86,8 @@ We enforce the style guide [PEP 8] with flake8, and [isort] for import sorting.
 $ black itdagene                # Fix code formatting
 $ tox -e isort -e flake8 -e yapf   # Verify code style
+
+$ make fixme # Fix all formatting
```
:heart_eyes:

odinuge commented:
(also, no more `styl` :heart_eyes:)

LudvigHz commented:
_complete overhaul_ is probably an overstatement. It's really just an overhaul of the js assets and a bump of adminLTE to v3, among some very minor tweaks (i.e. removed some dead links, added some buttons and options to delete pages and events).

But here are some screens of the "new" look:
![company](https://user-images.githubusercontent.com/42850232/82908529-02bf7500-9f60-11ea-98ef-f1c582327c85.png)
![pages](https://user-images.githubusercontent.com/42850232/82908538-04893880-9f60-11ea-9fa1-018557cc6a79.png)
